docs/workflow/building/libraries/README.md

```diff
@@ -193,30 +187,30 @@ build.cmd clr.corelib+clr.nativecorelib+libs.pretest -rc Release
 
 When this `System.Private.CoreLib` will be built in Release mode, then it will be crossgen'd and we will update the testhost to the latest version of corelib. 
 
-You can use the same workflow for mono runtime by using `mono.corelib+libs.pretest` subsets.
+You can use the same workflow for mono runtime by using `mono.corelib+libs.pretest` subsets and specifying `/p:RuntimeFlavor=Mono`.
```

Comment:
we don't need `RuntimeFlavor` in this case since it will be auto-detected when we're not building the `clr` subset.